docs/theme/bower_components_static/bourbon/.bower.json

```diff
@@ -23,4 +23,4 @@
   "_source": "https://github.com/thoughtbot/bourbon.git",
   "_target": "4.0.2",
   "_originalSource": "bourbon"
-}
\ No newline at end of file
+}
```

Comment:
Not convinced we should be auto-formatting these files; they look like they were copied from somewhere.